string user_id
int32 key_id
string scale_type
---
bool success